### PR TITLE
Update/import

### PR DESCRIPTION
**Pull request by RyanJField**
Implements Rio to allow uploading via xlsx and better support for csv files.

### CODE REVIEW

File: app.R [hunk 1]
```diff
@@ -269,7 +268,7 @@ ui <- navbarPage(title = "MetaDTA: Diagnostic Test Accuracy Meta-analysis",
                                            assessment data is not included in the file the covariates should be entered starting at the", tags$strong("seventh"), "column. If quality assessment
                                            data is included in the file the covariate data should be entered starting at the", tags$strong("fourteenth"), "column. Multiple covariates can be entered."),
                                          br(),
-                                         p("Note: Excel files should be saved in csv format and the separator option 'comma' selected for upload."),
+                                         p("Both Excel (.xlsx) and Comma Seperated Value (.csv) files are accepted."),
```
Comment (nabury) [inline app.R:271]: This would be better moved to under the "Please select a file" button as it is hidden in all the text

File: app.R [hunk 2]
```diff
@@ -655,7 +654,8 @@ server <- function(input, output) {
       }
     }
     else
-      a <- read.table(file = file1$datapath, sep = input$sep, header = input$header, stringsAsFactors = FALSE)
+      a <- rio::import(file1$datapath, header = input$header, stringsAsFactors = FALSE)
```
Comment (nabury) [inline app.R:657]: Does this need any tests (or are there some example files I can use to test manually?)

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-**MetaDTA v2.0.5**
+**MetaDTA v2.1.0**
```
Comment (nabury) [inline README.md:1]: Update link to crsu.shinyapps.io/MetaDTA (I updated the sidebar in the repo)

File: app.R (hunk 1 above)
Comment (RyanJField) [reply app.R]: I have moved this to the file input help.

File: README.md (hunk 3 above)
Comment (RyanJField) [reply README.md:1]: Thank you, updated.

Comment (RyanJField) [reply README.md:1]: I have also removed the link to metadta v1, as this no longer exists (it just redirects to this version),

File: app.R (hunk 2 above)
Comment (RyanJField) [reply app.R:656]: I believe you have already tested this manually, but the default files can be opened in excel and saved as .xlsx

**Review by nabury — APPROVED** [review]
I've edited the latest update text so it's accurate

**Pull request merged**